src/runtime/data/kata-collect-data.sh

```diff
@@ -803,6 +802,17 @@ read_osbuilder_file()
 	cat "$file"
 }
 
+show_latest_kata_version()
+{
+	# kata-check will not run network checks as root user, but this script requires root to run
+	local latest_version=$(sudo -u nobody kata-runtime kata-check --check-version-only 2>/dev/null || echo "error")
+	if [ "$latest_version" = "error" ]; then
+		msg "Unable to check for latest version. Check if system is connected"
+	else
+		msg "$latest_version"
```

Comment:
In case of no latest version, it will print something like this:

```
Running `kata-collect-data.sh` version `2.5.0-rc0 (commit 4f49423c91285fce6399da034c2867948b4dc616-dirty)` at `2022-08-08.18:42:57.977553756+0000`.
No newer release available
```

What does it print in case of a newer release is found?

Reply:
]"Newer %s release available: %s (url: %v, date: %v)\n", desc, details.version, details.url, details.date)"

If there is a newer release it should call showLatestRelease, which outputs the above formatted string.